ndc-models/src/relational_query/notes.md

```diff
@@ -0,0 +1,14 @@
+# Changes
```

Comment:
Did you mean to include this file? It looks a bit out of date but maybe it could become the RFC.

Reply:
No, sorry, this should be deleted, this was just my personal notes.